library/core/src/intrinsics.rs

```diff
@@ -3420,7 +3439,7 @@ pub const unsafe fn copy<T>(src: *const T, dst: *mut T, count: usize) {
                 align: usize = align_of::<T>(),
             ) =>
             ub_checks::is_aligned_and_not_null(src, align)
-                && ub_checks::is_aligned_and_not_null(dst, align)
+            && ub_checks::is_aligned_and_not_null(dst, align)
```

Comment:
Why is this being reformatted?

Reply:
Good question. My IDE is very opinionated. LOL

We need to enable Rusty's expected formatting to this repo so we don't diverge too much on code formatting.